rskj-core/src/main/java/co/rsk/peg/BridgeSupport.java

```diff
@@ -1351,7 +1351,7 @@ public Integer getRetiringFederationThreshold() {
             return null;
         }
 
-        List<BtcECKey> publicKeys = retiringFederation.getPublicKeys();
+        List<BtcECKey> publicKeys = retiringFederation.getBtcPublicKeys();
 
         if (index < 0 || index >= publicKeys.size()) {
             throw new IndexOutOfBoundsException(String.format("Retiring federator index must be between 0 and {}", publicKeys.size() - 1));
```

Comment:
the {} should be %s in String.format. I think this kind of interpolation is in logs only in this proyect